site/en/blog/progress-in-the-privacy-sandbox-2021-10/index.md

```diff
@@ -206,18 +206,17 @@ Advertising Technology Community Group)](https://www.w3.org/community/patcg/)**.
 ## Measure digital ads
 
 As the companion to displaying ads without cross-site tracking, we need
-privacy-preserving mechanisms to enable measuring the effectiveness of those
-ads.
+privacy-preserving mechanisms to measure the effectiveness of those ads.
 
 ### Attribution Reporting API
 
 The **[Attribution Reporting
-API](/docs/privacy-sandbox/attribution-reporting/)** enables functionality to
+API](/docs/privacy-sandbox/attribution-reporting/)** gives you the ability to
 measure events on one site, like clicking or viewing an ad, that lead to a
-conversion on another site—without enabling cross-site tracking.
+conversion on another site&mdash;without enabling cross-site tracking.
 
-We would like to continue testing on the Attribution Reporting API and we are
-planning on **[extending the origin
+We would like to continue testing on the Attribution Reporting API and we plan
+on **[extending the origin
 trial](https://groups.google.com/a/chromium.org/g/blink-dev/c/DdjaFmsb4fA)**
```

Comment:
@jpchase Do you have a token URL for the extension? The old OT is on the dashboard, but the new one isn't. This article should link to it.

Reply:
For that intent, the existing trial for Conversion Measurement was approved for an extension, not a new trial. You can see the end is now in Chrome 97: https://developer.chrome.com/origintrials/#/view_trial/3411476717733150721. I think the text here should be updated to say the trial has been extended, and use the existing OT url.

Reply:
@rowan-m I can't add this through a suggestion because it's not part of text you touched. Can you please update this?

Reply:
I updated this URL, but it wasn't true that the trial was extended at the time this was published. With a November update coming, that will be the better time to reference the change.